packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -163,6 +164,20 @@ abstract class Logger {
     bool? wrap,
   });
 
+  /// Display the [message] inside a box.
+  /// For example, this is the generated output:
```

Comment:
dartdoc needs a blank line between the first sentence and the rest (not that it matters in flutter_tools, but it's the principle of the thing :-) )

Reply:
done